src/com/google/javascript/jscomp/parsing/IRFactory.java

```diff
@@ -1792,6 +1792,16 @@ Node processUnaryExpression(UnaryExpressionTree exprNode) {
               msg,
               sourceName,
               operand.getLineno(), 0);
+        } else if ((type == Token.INC || type == Token.DEC)
```

Comment:
These two snippets look pretty similar. Maybe we can factor them out into a separate method to avoid duplication?